app/build.gradle

```diff
@@ -163,6 +164,7 @@ android {
             buildConfigField "String", "EVENTLOG_WIKI", "\"commonswiki\""
             buildConfigField "String", "SIGNUP_LANDING_URL", "\"https://commons.m.wikimedia.beta.wmflabs.org/w/index.php?title=Special:CreateAccount&returnto=Main+Page&returntoquery=welcome%3Dyes\""
             buildConfigField "String", "SIGNUP_SUCCESS_REDIRECTION_URL", "\"https://commons.m.wikimedia.beta.wmflabs.org/w/index.php?title=Main_Page&welcome=yes\""
+            buildConfigField "String", "FORGOT_PASSWORD_URL", "\"https://commons.wikimedia.org/wiki/Special:PasswordReset\""
```

Comment:
Thanks for moving this constant to the build but it still needs a final tweak to point at the correct forgot password URL for the "beta" flavor - https://commons.wikimedia.beta.wmflabs.org/wiki/Special:PasswordReset